### DISCUSSION TITLE
Export excel [Sep 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6393] [Discord Threads]
> 🙋‍♂️ **Developer**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@964756055609458719> There is an option for download CSV. Only table currently working with will be downloaded as a CSV file.  
![image0](https://cdn.discordapp.com/attachments/993782464298373180/993805406734385232/Screenshot_2022-07-05_at_2.37.35_PM.png)  
#  
> 🙋‍♂️ **Developer**  
#### Yes I saw that but need to export in excel format  
#  
> 👨‍🦲 **dstala**  
#### Download in Excel format is not supported. Individual CSV filed downloaded per table needs to be imported onto excel (using spreadsheet/ MS excel) manually.  
#  
> 🙋‍♂️ **Developer**  
#### can we add that functionality in noco DB platform using some customization ?  
#  
> 👨‍🚀 **wingkwong**  
#### please raise a feature request in github  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### btw. export in excel is now possible.  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I just did an export, expected over 500 rows but only got 300. Is that the standard?  
#  
> 👨‍🚀 **wingkwong**  
#### You should be expected to get all the records unless you've set some filters. If you haven't set filters, please create a issue.  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### Created the bug report  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### We have some stuff in our pipeline, but if we can, we’ll try to fix it at the end of August  
#  
> 👨‍🚀 **wingkwong**  
#### can u show the issue number / link ?  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### https://github.com/nocodb/nocodb/issues/6116  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I just read the last comment. We didn't roll out the 0.109.6 version because I found a bug in our testing instance. I'll have to check in my chat history what was the topic since I can't recall right now.    
#### In the test instance I have a table with over 3000 entries, when exporting to excel I only get 400 entries in 1 file, no filters applied.    
#### Then I get a browser request to download more files,  then it exports as many files as it needs, each file with 400 entries. I'll add this to the bug report  
#  
> 👨‍🚀 **wingkwong**  
#### so after allowing multi download - the total number of records are correct (?) just split in multiple files  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### yes  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### 8 tables with 400 entries each and one table with 323, for a total of 3523 entries  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I can work like that for some time. Others will have issues. As I said, end of August we could have some capacity to look into it. Ofc if it's fix before that, we'll test it again  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/993782464298373180)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
